customer-node/service/flightsService.js

```diff
@@ -0,0 +1,5 @@
+const { findFlights } = require('../data_access/flightda');
+
+const getFlights = (source, destination, date) => findFlights(source, destination, date);
```

Comment:
Maybe check if source and destination exist in DB before querying for flights?

Reply:
done